source/language/types.rst

```diff
@@ -267,6 +267,38 @@ are all of type ``float[64]``.
 
 Note that `e` is a valid identifier. `e/E` are also used in scientific notation where appropriate.
 
+Mathematical functions available for constant initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to simple arithmetic functions used in expressions initializing constants,
+OpenQASM 3 offers the following built-in mathematical operators followed by
+their argument expression in parentheses:
+
+.. container::
+   :name: tab:built-in-math
+
+   .. table:: Built-in mathematical functions in OpenQASM3.
+
+      +--------+
+      | arccos |
+      +--------+
+      | arcsin |
+      +--------+
+      | arctan |
+      +--------+
+      | cos    |
+      +--------+
+      | exp    |
+      +--------+
+      | ln     |
+      +--------+
+      | sin    |
+      +--------+
+      | sqrt   |
```

Comment:
`sqrt(float) -> float` (error on negative) and `sqrt(complex) -> complex`, and both should comment that they take the principal root, I would think.